daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -1551,6 +1551,73 @@ class RazerViper8KHz(__RazerDevice):
 
     POLL_RATES = [125, 500, 1000, 2000, 4000, 8000]
 
+class RazerViperMiniSEWireless(__RazerDevice):
```

Comment:
Normally for wired/wireless devices we make the wireless one inherit the wired one and just extend extra methods if needed or overwrite e.g. `USB_PID`. Reduces code duplication since it's a single device really

Reply:
I see. The daemon hardware config reflect should the correct way of doing things now.